### ISSUE TITLE
Unable to render equations after update to 2.2.1

### ISSUE THREAD

**Issue by nmanumr** [squidfunk/mkdocs-material#627]
I just updated my mkdocs-material package using cammand
```
pip install --upgrade mkdocs-material
```
Everything was working fine before the update. but after update equations seems not to render properly.

Here is an image before update
![image](https://user-images.githubusercontent.com/19629102/33696058-3f7dd338-db22-11e7-9a34-de66c5c42c4c.png)

And here is after update
![image](https://user-images.githubusercontent.com/19629102/33696087-5e3cecd2-db22-11e7-8798-3a2254fbd3aa.png)

I am using katex instead of mathjax and here is my markdown code:
```markdown
An algebraic expression consisting of two terms such as $a+x$, $ax+b$ etc is called binomial or a binomial expression.

## Binomial Formula
The following expressions are valid for any $n\in Z^n$. (n is called index)

$$ (a+b)^n = a^n +\frac{n}{1!} a^{n-1} b^1+ \frac{n(n-1)}{2!} a^{n-2} b^2 + \frac{n(n-1)(n-2)}{3!} a^{n-3} b^3 +...+ b^n $$

$$ (a+b)^n = ^nC_0 a^n + ^nC_1 a^{n-1} b^1 + ^nC_2 a^{n-2} b^2 +...+ ^nC_r a^{n-r} b^r +...+^nC_n a^{n-n} b^n $$

$$ (a+b)^n = \displaystyle\sum_{r=0}^n \binom{n}{r} a^{n-r} b^r $$
```

### Steps to reproduce the bug
May be the code provided above can reproduce the issue with the configurations provided below.

### Package versions

* Python: 2.7.14
* MkDocs: 0.17.2
* Material: 2.2.1

### Project configuration

``` yaml
# Project information
site_name: 'Entry Test Notes'
site_description: 'Entry Test notes for FUNG enrty test prepration'
site_author: 'Nauman Umer'
site_url: 'https://nmanumr.github.io/entry-test-notes'

# Repository
repo_name: 'nmanumr/Fung-notes'
repo_url: 'https://www.github.com/nmanumr/nmanumr.github.io'
edit_uri: 'blob/master/entry-test-notes/docs/'

# Configuration
theme:
  name: 'material'
  feature:
    tabs: true

#Pages
pages:
## a lot of pages ...

# Extensions
markdown_extensions:
  - footnotes
  - toc:
      permalink: true
  - pymdownx.arithmatex
  - pymdownx.betterem:
      smart_enable: all
  - pymdownx.caret
  - pymdownx.critic
  - pymdownx.details
  - pymdownx.emoji:
      emoji_generator: !!python/name:pymdownx.emoji.to_svg
  - pymdownx.inlinehilite
  - pymdownx.magiclink
  - pymdownx.mark
  - pymdownx.smartsymbols
  - pymdownx.superfences
  - pymdownx.tasklist:
      custom_checkbox: true
  - pymdownx.tilde
  - admonition

extra_css:
  - 'https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-alpha1/katex.min.css'
  - 'css/index.css'

extra_javascript:
  - 'https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-alpha1/katex.min.js'
  - 'https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-alpha1/contrib/auto-render.min.js'
  - 'js/index.js'
```

### System information
* OS: Ubuntu 17.10
* Browser: Chrome 62.0.3202.94

**Comment by facelessuser** [comment]
@nmanumr, this is not a Material issue, but would be a pymdown-extensions issue.  Version 4.6.0 recently simplified arithmatex logic to output in MathJax script format (wrapped in a script).  The reason why this wasn't done before is that the script format won't show a fallback preview while loading.  But I recently discovered how to do previews with the script format, so I migrated to that to have one output format.

In short, Arithmatex was originally written for MathJax, so that was the only library I check against.  I was either unaware or have since forgotten that libraries like KaTeX are available to do this as well.  So this change was backwards compatible for MathJax, which I test for, but not for anything else.

With that said, I see no reason why I can't add an option to bring back the plain format (non-script) you are used to.  Create an issue over at the [pymdown-extensions issue page](https://github.com/facelessuser/pymdown-extensions/issues) so I don't forget.
